### PR TITLE
Update homebank to 5.0.0

### PR DESCRIPTION
**Pull request by TomK32**
Only change necessary was to use gtk+3.

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Can you adjust the `chmod` line (not your issue I know but just cleaning things up as we go along): http://bot.brew.sh/job/Homebrew%20Pull%20Requests/21114/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_homebank/

**Comment by TomK32** [comment]
right, only mountainlion and something about libofx? The previous error took me 30 minutes to dig and fix, can I get a little hint this time?

File: Library/Formula/homebank.rb [hunk 1]
```diff
@@ -26,7 +26,7 @@ def install
     args << "--with-ofx" if build.with? 'libofx'
 
     system "./configure", *args
-    system "chmod +x ./install-sh"
+    chmod("+x", "install-sh")
```
Comment (mikemcquaid) [inline Library/Formula/homebank.rb]: `chmod 0755, "install.sh"`

Comment (TomK32) [inline Library/Formula/homebank.rb]: Personally I'd change it to `u+x`, never been much a fan of the octal numbers.

Anyways, this is not why I does fail the process for mountain lion.

Comment (jacknagel) [inline Library/Formula/homebank.rb]: This *is* what is causing the failure:

```
Error: can't convert String into Integer
Please report this bug:
    http://git.io/brew-troubleshooting
/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/fileutils.rb:1223:in `chmod'
/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/fileutils.rb:1223:in `chmod'
/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/fileutils.rb:876:in `chmod'
/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/fileutils.rb:875:in `each'
/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/fileutils.rb:875:in `chmod'
/usr/local/Library/Formula/homebank.rb:29:in `install'
/usr/local/Library/Homebrew/build.rb:136:in `install'
/usr/local/Library/Homebrew/formula.rb:410:in `brew'
/usr/local/Library/Homebrew/formula.rb:768:in `stage'
/usr/local/Library/Homebrew/resource.rb:91:in `unpack'
/usr/local/Library/Homebrew/extend/fileutils.rb:21:in `mktemp'
/usr/local/Library/Homebrew/resource.rb:88:in `unpack'
/usr/local/Library/Homebrew/resource.rb:81:in `stage'
/usr/local/Library/Homebrew/formula.rb:766:in `stage'
/usr/local/Library/Homebrew/formula.rb:406:in `brew'
/usr/local/Library/Homebrew/build.rb:114:in `install'
/usr/local/Library/Homebrew/build.rb:185
```

FileUtils.chmod in Ruby 1.8 does not accept string representation of the mode.

Comment (TomK32) [inline Library/Formula/homebank.rb]: ah.

File: Library/Formula/homebank.rb [hunk 2]
```diff
@@ -26,7 +26,7 @@ def install
     args << "--with-ofx" if build.with? 'libofx'
 
     system "./configure", *args
-    system "chmod +x ./install-sh"
+    chmod(0755 "install-sh")
```
Comment (jacknagel) [inline Library/Formula/homebank.rb]: You're missing a comma after the 0755 (drop the parens also)

**Comment by jacknagel** [comment]
homebank has been updated in the meantime.

**Pull request closed without merging**